mycroft/configuration/config.py

```diff
@@ -235,28 +245,36 @@ def load_config_stack(configs=None, cache=False, remote=True):
         Returns:
             (dict) merged dict of all configuration files
         """
+
+        # system administrators can define different constraints in how
+        # configurations are loaded
+        system_conf = _get_system_constraints()
+        protected_keys = system_conf.get("protected_keys") or {}
+        protected_remote = protected_keys.get("remote") or []
+        protected_user = protected_keys.get("user") or []
+        skip_user = system_conf.get("disable_user_config", False)
+        skip_remote = system_conf.get("disable_remote_config", False)
+
+        # This includes both the user config and
+        # /etc/xdg/mycroft/mycroft.conf
+        xdg_locations = get_xdg_config_locations()
+
         if not configs:
             configs = [LocalConf(DEFAULT_CONFIG),
                        LocalConf(SYSTEM_CONFIG)]
-            if remote:
-                   configs.append(RemoteConf())
-
-            if is_using_xdg():
-                # deprecation warning
-                if isfile(OLD_USER_CONFIG):
-                    _log_old_location_deprecation(OLD_USER_CONFIG)
+            if not skip_remote and remote:
+                configs.append(RemoteConf())
+            if not skip_user:
+                if is_using_xdg():
+                    # deprecation warning
+                    if isfile(OLD_USER_CONFIG):
+                        _log_old_location_deprecation(OLD_USER_CONFIG)
+                        configs.append(LocalConf(OLD_USER_CONFIG))
+                    configs += [LocalConf(p) for p in xdg_locations]
+                else:
+                    # just load the pre defined old locations
                     configs.append(LocalConf(OLD_USER_CONFIG))
-
-                # This includes both the user config and
-                # /etc/xdg/mycroft/mycroft.conf
-                configs += [LocalConf(p) for p in get_xdg_config_locations()]
-
-                configs.append(Configuration.__patch)
-            else:
-                # just load the pre defined locations
-                configs += [LocalConf(OLD_USER_CONFIG),
-                            Configuration.__patch]
-
+            configs.append(Configuration.__patch)
```

Comment:
it looks like this was under the if on new L268 (ref old L254)

did the logic flow change? it looks the same up to here.

Reply:
that one is always loaded last. previously it was inside the `if` and also inside the `else`, just moved it to remove the duplicated logic